2022-03-13 20:48:06,080 INFO    MainThread:30328 [wandb_setup.py:_flush():75] Loading settings from /nethome/mrungta8/.config/wandb/settings
2022-03-13 20:48:06,080 INFO    MainThread:30328 [wandb_setup.py:_flush():75] Loading settings from /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning/wandb/settings
2022-03-13 20:48:06,080 INFO    MainThread:30328 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-13 20:48:06,081 INFO    MainThread:30328 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'Mukund/RE_improved_baseline/meta_learning/train_tacred.py', 'program': 'train_tacred.py'}
2022-03-13 20:48:06,081 INFO    MainThread:30328 [wandb_init.py:_log_setup():386] Logging user logs to /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning/wandb/run-20220313_204806-satl81jh/logs/debug.log
2022-03-13 20:48:06,081 INFO    MainThread:30328 [wandb_init.py:_log_setup():387] Logging internal logs to /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning/wandb/run-20220313_204806-satl81jh/logs/debug-internal.log
2022-03-13 20:48:06,081 INFO    MainThread:30328 [wandb_init.py:init():420] calling init triggers
2022-03-13 20:48:06,081 INFO    MainThread:30328 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-03-13 20:48:06,081 INFO    MainThread:30328 [wandb_init.py:init():471] starting backend
2022-03-13 20:48:06,081 INFO    MainThread:30328 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-13 20:48:06,094 INFO    MainThread:30328 [backend.py:ensure_launched():219] starting backend process...
2022-03-13 20:48:06,102 INFO    MainThread:30328 [backend.py:ensure_launched():225] started backend process with pid: 30476
2022-03-13 20:48:06,104 INFO    MainThread:30328 [wandb_init.py:init():480] backend started and connected
2022-03-13 20:48:06,111 INFO    MainThread:30328 [wandb_init.py:init():550] updated telemetry
2022-03-13 20:48:06,137 INFO    MainThread:30328 [wandb_init.py:init():581] communicating current version
2022-03-13 20:48:07,223 INFO    MainThread:30328 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-13 20:48:07,223 INFO    MainThread:30328 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-13 20:48:07,324 INFO    MainThread:30328 [wandb_init.py:init():624] starting run threads in backend
2022-03-13 20:48:09,092 INFO    MainThread:30328 [wandb_run.py:_console_start():1827] atexit reg
2022-03-13 20:48:09,093 INFO    MainThread:30328 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2022-03-13 20:48:09,095 INFO    MainThread:30328 [wandb_run.py:_redirect():1706] Redirecting console.
2022-03-13 20:48:09,103 INFO    MainThread:30328 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-13 20:48:09,104 INFO    MainThread:30328 [wandb_init.py:init():651] run started, returning control to user process
2022-03-13 20:51:04,403 INFO    MainThread:30328 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 1
2022-03-13 20:51:04,407 INFO    MainThread:30328 [wandb_run.py:_restore():1769] restore
2022-03-13 20:51:06,694 INFO    MainThread:30328 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 949
}

2022-03-13 20:51:06,903 INFO    MainThread:30328 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 949
}

2022-03-13 20:51:07,010 INFO    MainThread:30328 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 949
}

2022-03-13 20:51:07,522 INFO    MainThread:30328 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 7062
}

2022-03-13 20:51:07,626 INFO    MainThread:30328 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 7090
}

2022-03-13 20:51:07,729 INFO    MainThread:30328 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7090
  total_bytes: 7090
}

2022-03-13 20:51:07,833 INFO    MainThread:30328 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7090
  total_bytes: 7090
}

2022-03-13 20:51:07,936 INFO    MainThread:30328 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7090
  total_bytes: 7090
}

2022-03-13 20:51:08,039 INFO    MainThread:30328 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7090
  total_bytes: 7090
}

2022-03-13 20:51:08,289 INFO    MainThread:30328 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7090
  total_bytes: 7090
}

2022-03-13 20:51:08,513 INFO    MainThread:30328 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7090
  total_bytes: 7090
}
local_info {
}

2022-03-13 20:51:10,037 INFO    MainThread:30328 [wandb_run.py:_append_files():2194] logging synced files
